python/ray/util/placement_group.py

```diff
@@ -109,11 +114,17 @@ def bundle_count(self):
         return len(self.bundle_cache)
 
     def _get_none_zero_resource(self, bundle: List[Dict]):
-        # Any number between 0-1 should be fine.
-        INFEASIBLE_TASK_SUPPRESS_MAGIC_NUMBER = 0.0101
+        # Set a mock value to schedule a dummy task.
+        MOCK_VALUE = 0.001
         for key, value in bundle.items():
             if value > 0:
-                value = INFEASIBLE_TASK_SUPPRESS_MAGIC_NUMBER
+                if key == "memory":
+                    # Make sure the memory resource can be
+                    # transformed to memory unit.
+                    to_memory_units(value, True)
```

Comment:
Remove this line?

Reply:
Can you just do `value = MEMORY_RESOURCE_UNIT_BYTES if key == "memory" else MOCK_VALUE `

Reply:
I think we still need `to_memory_units(value, True)`, this is a pre-check, because users may set memory resources with `0.01` in bundles, but I will move this to the method of `validate bundles`.